components/ParameterForm.vue

```diff
@@ -65,13 +69,14 @@
       <CButton
         id="run-button"
         color="primary"
-        :size="largeScreen ? 'lg' : undefined"
+        :size="screenIsLarge ? 'lg' : undefined"
         type="submit"
+        :disabled="formSubmitting || !pageMounted"
```

Comment:
a loading icon could be nice also whilst `formSubmitting`

Reply:
Yep, I put a spinner into the button. Not that it hangs around long enough for a human to see it!

Reply:
(maybe it will though when the subsequent page takes actual time to load)